src/RoslynPad.Roslyn/SymbolDisplayPartExtensions.cs

```diff
@@ -12,6 +13,9 @@ public static class SymbolDisplayPartExtensions
     {
         private const string LeftToRightMarkerPrefix = "\u200e";
 
+        public static Action<SymbolDisplayPartKind, Run> StyleRunFromSymbolDisplayPartKind { get; set; }
```

Comment:
Instead of this static delegate, I would rather have an interface to handle this. Something like

```csharp
interface ITaggedTextFormatter
{
    void Format(Run run, TaggedText taggedText);
}
```

Then we can have a default instance of this with the default colors.